### PR TITLE
E2603: Update View Submissions Page

### PR DESCRIPTION
**Pull request by CamBhat**
E2603: Implement ViewSubmissions frontend

- ViewSubmissions page updated to include loading and new content utilizing API call
- ViewSubmissions Test file updated to include edge cases
- Test file ran and confirmed to pass all tests

## Summary by CodeRabbit

* **New Features**
  * Full assignment submission workspace: folder-based uploads, hyperlink submissions, breadcrumbs and per-item actions
  * Submissions viewer for instructors with real submissions, direct artifact open/download and per-artifact progress
  * "Assign grade" workflow added for post-deadline grading

* **Improvements**
  * Rebuilt UI with cohesive styling, responsive layout and improved tables/controls
  * Better loading, empty and error states with global alerts and action-specific spinners
  * Updated assignment routes and navigation flow

* **Tests**
  * Added comprehensive tests for submissions listing and service behaviors

### CODE REVIEW

**Comment by CamBhat** [comment]
Moved to draft to work through final project:

Intending on
- Updating page to utilize SubmissionContent if needed
- Updating test cases to ensure submitted files are being pulled in properly

FYI @mcarthur-reece feel free to add more here if needed

File: src/services/SubmittedContentService.ts [hunk 1]
```diff
@@ -1,296 +1,278 @@
...
-    }
+    const formData = new FormData();
+    formData.append("id", participantId);
+    formData.append("uploaded_file", file);
+    formData.append("current_folder[name]", this.normalizeFolder(currentFolder));
+
+    const response = await axiosClient.post("/submitted_content/submit_file", formData);
+
+    return response.data;
   }
```
[the remark this answers is not in the document]
Comment (CamBhat) [reply src/services/SubmittedContentService.ts:116]: Resolved in latest commit
